sdk/storage/Azure.Storage.Blobs.Batch/api/Azure.Storage.Blobs.Batch.netstandard2.0.cs

```diff
@@ -1,9 +1,17 @@
+namespace Azure.Storage.Blobs.Models
+{
+    public enum BlobDeleteType
+    {
+        Permanent = 0,
+    }
+}
 namespace Azure.Storage.Blobs.Specialized
 {
     public partial class BlobBatch : System.IDisposable
     {
         protected BlobBatch() { }
         public BlobBatch(Azure.Storage.Blobs.Specialized.BlobBatchClient client) { }
+        public bool IsContainerScoped { get { throw null; } }
```

Comment:
should this be internal ?

Reply:
I think it could provide useful information to the user.  I'm open to removing it.

Reply:
I'd hide it for now. Until someone actually needs it. I'm worried "scoping" is going to evolve beyond just container - in that case modeling this as flag won't be great.

Reply:
If this comes back maybe we should add enum BatchScope { Account, Container } and have property `Scope`

Reply:
Fixed.